tests/test_smile.py

```diff
@@ -647,14 +647,23 @@ async def tinker_regulation_mode(smile):
     async def tinker_max_boiler_temp(smile):
         """Change max boiler temp setpoint to test functionality."""
         new_temp = 60.0
+        dev_id = None
         _LOGGER.info("- Adjusting temperature to %s", new_temp)
         for test in ["maximum_boiler_temperature", "bogus_temperature"]:
             try:
-                await smile.set_number_setpoint(test, new_temp)
+                await smile.set_number_setpoint(test, dev_id, new_temp)
                 _LOGGER.info("  + worked as intended")
             except pw_exceptions.PlugwiseError:
                 _LOGGER.info("  + failed as intended")
 
+    @staticmethod
+    async def tinker_temp_offset(smile, dev_id):
+        """Change temperature_offset to test functionality."""
+        new_offset = 1.0
+        _LOGGER.info("- Adjusting temperature offset to %s", new_offset)
```

Comment:
we can only set unless we manually create a mock replying it changed, right?

Reply:
When looking at the test-output:
```
INFO     test_smile:test_smile.py:663 - Adjusting temperature offset to 1.0
INFO     aiohttp.access:web_log.py:206 127.0.0.1 [05/Aug/2023:07:06:14 +0000] "PUT /core/appliances;id=7ffbb3ab4b6c4ab2915d7510f7bf8fe9/offset;type=temperature_offset HTTP/1.1" 202 164 "-" "Python/3.10 aiohttp/3.8.5"
INFO     test_smile:test_smile.py:666   + worked as intended
INFO     test_smile:test_smile.py:663 - Adjusting temperature offset to 1.0
INFO     test_smile:test_smile.py:669   + failed as intended
```
no extra mocking needed.